corehq/apps/toggle_ui/templates/toggle/edit_flag.html

```diff
@@ -6,10 +6,9 @@
 {% block title %}{% trans "Edit Feature Flag: " %}{{ toggle_meta.label }}{% endblock %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
-    <script src="{% static 'hqwebapp/js/main.js' %}"></script>
-    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
-    <script src="{% static 'formdesigner/js/lib/underscore-1.5.2.js' %}"></script>
+    <script src="{% new_static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
+    <script src="{% new_static 'hqwebapp/js/main.js' %}"></script>
+    <script src="{% new_static 'formdesigner/js/lib/underscore-1.5.2.js' %}"></script>
```

Comment:
actually i think underscore is always loaded on the page so we could probably get rid of it entirely